src/Dev-8/CreateMatrix.java

```diff
@@ -0,0 +1,41 @@
+import java.util.Scanner;
+
+/**
+ * Class for create matrix: double[][]
+ */
+public class CreateMatrix {
+    final String ENTER_NUMBER_OF_COLUMNS = "Enter number of columns: ";
+    final String ENTER_NUMBER_OF_ROWS = "Enter number of rows: ";
+    private int rows;
+    private int colums;
+
+    /**
+     * @return created matrix
+     */
+    public double[][] create() {
+        Scanner in = new Scanner(System.in);
+        System.out.println(ENTER_NUMBER_OF_ROWS);
+        rows = in.nextInt();
+        System.out.println(ENTER_NUMBER_OF_COLUMNS);
+        colums = in.nextInt();
+        double[][] maxrix = new double[rows][colums];
```

Comment:
No check for rows and colums validity, can be negative, checkArgumentsIsLegal has to be before line 21